test/KernelStakes.test.js

```diff
@@ -0,0 +1,131 @@
+import assertRevert from "./helpers/assertRevert";
+
+const KernelStakes = artifacts.require('KernelStakes');
+const shouldBehaveLikeOwnable = require('zos-core/test/ownership/Ownable.behavior');
+
+contract('KernelStakes', ([_, stakeOwner, someone, instance]) => {
+
+  beforeEach(async function () {
+    this.stakes = await KernelStakes.new({ from: stakeOwner })
+  })
+
+  describe('ownership', function () {
+    beforeEach(function () {
+      this.ownable = this.stakes
+    })
+
+    shouldBehaveLikeOwnable(stakeOwner, someone)
+  })
+
+  describe('stake', function () {
```

Comment:
Let's add a test on: 
- Staking twice from the same user for the same stuff, and checking that the values are added
- Staking for the same stuff from two different users, and checking that totalStakedFor is the sum